.swiftpm/xcode/xcshareddata/xcschemes/NiceComponents.xcscheme

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I dont think we need to include this file - can we remove it please?

Reply:
Yeah, I was wondering that as well. Is there an easy way to remove a file like this? This is where I tend to break PRs. I assume it will need to be added to the `.gitignore` as well?

Reply:
Since you haven't merged anything yet and we don't care if the file is in the commit history, just that it doesn't make it into `main`, it should be pretty easy to just Google a quick tutorial on how to do this. If you try a couple things and it doesn't work out and you're stuck let me know and we can walk through it together.